recipes-support/amazon-ssm-agent/amazon-ssm-agent_git.bb

```diff
@@ -0,0 +1,70 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = " \
+    file://src/${GO_IMPORT}/LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+    "
+
+GO_IMPORT = "github.com/aws/amazon-ssm-agent"
+SRC_URI = "git://${GO_IMPORT};branch=mainline;protocol=https"
+
+PV = "3.1.0.0+git${SRCPV}"
+SRCREV = "232b1d9a483b0404530798bf412409d23aecf630"
+
+S = "${WORKDIR}/git"
+
+inherit go systemd
+
+SYSTEMD_AUTO_ENABLE = "enable"
+SYSTEMD_SERVICE:${PN} = "amazon-ssm-agent.service"
+
+do_configure () {
```

Comment:
I am not sure what this is supposed to be doing, but if you're trying to skip do_configure, the usual pattern is:

```
do_configure[noexec] = "1"
```